javascript/stimulus_reflex.js

```diff
@@ -34,8 +34,14 @@ let actionCableSubscriptionActive = false
 // A dictionary of all active Reflex operations, indexed by reflexId
 window.reflexes = {}
 
-// Indicates if we should log calls to stimulate, etc...
-let debugging
+// Debug state convenience functions
+const getDebug = () => {
+  return sessionStorage.getItem('stimulus-reflex-debugging') === 'true' || false
```

Comment:
Do we need the explicit `|| false` here?

Reply:
Probably not. Being overly cautious again 😁